### ISSUE TITLE
Can you still request permissions on v4 Android?

### ISSUE THREAD

**Issue by ChrisMott97** [capacitor-community/contacts#89]
**Describe the bug**
The docs say there's a [requestPermissions()](https://capacitor-community.github.io/contacts/#/api?id=requestpermissions) method, but I can't seem to see a PluginMethod in the android source for this method? I was looking for it to be sure of what the behavior was as we're migrating from the old @ionic-native/contacts plugin 

I'm using Capacitor 4, and therefore v4-beta of this package

**Comment by tafelnl** [comment]
**TLDR:** upon using the plugin you will find that on Android (and iOS) it works exactly like the TypeScript definitions say it will. That means `requestPermissions` is in fact available.

**Detailed explanation:**
It is indeed correct that for Android you won't find a `requestPermissions` method in the source code, although this is the case for the iOS source code. That is because Capacitor (from v3 on) supports a new way of requesting permissions on iOS and Android with the use of some helpers provided by the Capacitor SDK. Because on Android most (if not all) permissions can be requested the same way, the Capacitor team decided to abstract it away completely. You can find some more details about this in the docs ([Android](https://capacitorjs.com/docs/v3/plugins/android#permissions)) ([iOS](https://capacitorjs.com/docs/v3/plugins/ios#permissions))
